### DISCUSSION TITLE
Selected word's duplicates highlight is very hard to see

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#7012] [Editor]
I think it would be a good idea to change the default settings for how the highlighting for duplicates of the selected word is handled by the editor.  In the current editor, this setting is so close to the default background that it is nearly invisible. (See the second occurrence of `to_plane` in the attached image).

Being able to highlight a particular word and then scrolling through your code to find any duplicates is a very helpful thing to do when debugging and it's not practical to do with the current default settings.

![image](https://github.com/godotengine/godot-proposals/assets/6724608/b739a65c-241f-4989-9603-3d2b3c48b2b2)

**Comment by AThousandShips** [comment]
This is a bug not a feature IMO, accessibility concerns

**Comment by blackears** [comment]
Should I file an issue?

**Reply by AThousandShips** [reply]
I'm unsure, but I'd consider this an issue rather than a feature improvement, as I definitely agree that is very hard to notice and I have very good eyesight

**Reply by blackears** [reply]
I've files an issue.  https://github.com/godotengine/godot/issues/77812

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot/issues/77812.
